mllib/src/main/scala/org/apache/spark/ml/regression/LinearRegression.scala

```diff
@@ -398,7 +422,8 @@ class LinearRegressionModel private[ml] (
 
   /**
    * Evaluates the model on a testset.
-   * @param dataset Test dataset to evaluate model on.
+    *
```

Comment:
I really don't know how this has happened. :)
I've fixed these. All the indentation should be in order now.